week-1/K-functions-parameters/exercise.js

```diff
@@ -1,9 +1,10 @@
 // Complete the function so that it takes input parameters
-function multiply() {
+
+function multiply(a, b) {
   // Calculate the result of the function and return it
+  return 3 * 4;
```

Comment:
Because you're multiplying `3` and `4` in the function here, any time you call it - no matter the parameters, it will **always** return 12.

How can you fix this, so when I call `multiply(2, 5)` it gives me 10?
Remember you should multiply the function parameters, not specific numbers